### ISSUE TITLE
Links in bulletpoints are not underlined

### ISSUE THREAD

**Issue by pranshukharkwal** [devbae/devbae-blog#24]
# Bug Report

## Description

If a link is present in a bullet point, the anchor text is not underlined. 
I am not sure if it is expected to be like that, but I think it would be better if there is an underline to indicate the link.

## Steps to Reproduce

Check out this blog post - https://devbae.tech/2020/06/10/introduction-to-graphql.html

## Expected Behaviour
In the using the IDEs section, the anchor text of the links in the bullet points should be underlined

## Actual Behaviour

The anchor texts are not underlined

## Settings

- Browser used: Chrome
- OS: Windows 10

## Screenshots, if any
![devbae-bug](https://user-images.githubusercontent.com/21126219/84776819-705e3e80-affe-11ea-83dd-88ad483d9c3d.png)

**Comment by aashish-ak** [comment]
This is a good observation @pranshukharkwal, will look into it :)

**Comment by Signior-X** [comment]
Can I do this?

**Comment by aashish-ak** [comment]
Sure @Signior-X

**Comment by Signior-X** [comment]
@aashish-ak 
Ek doubt hai regarding this issue.... yadi iske heesan se underline hone chahhiye to firr to isme bhi underline hoega na?

Yeh Jo list hai isme sidethrough vaale, yeh bhi underline hone chahiye firr to.....
To kya specific pehchaan honi chahiye ki list ke andar ke links ko underline hona chahiye ya nhi?

![Screenshot from 2020-06-27 03-00-49](https://user-images.githubusercontent.com/56730716/85902849-95368b00-b822-11ea-92ba-8c1c4c721406.png)

**Comment by aashish-ak** [comment]
Sirf jo lists blog content me hn whi underline honi chahiye to differentiate between normal text and links. Toc or sidebar me its obvious ki wo links hi honge.

**Comment by Signior-X** [comment]
Yes vahi keh raha hu jo isme bold letters ma list hai strikethrough kiye hue vo blog content ma aata hai..... To vobhi underline hojayega

**Comment by aashish-ak** [comment]
Yes, I think this is an issue, I don't know why the numbered list is allowing the underlines for links, and the unordered list is not. This is an issue with the markdown parser itself, you can see maybe if the blog content links can be identified separately and then you can underline them by using simple CSS, but if you can't, we will look for some other way to do this.

**Comment by Signior-X** [comment]
> Yes, I think this is an issue, I don't know why the numbered list is allowing the underlines for links, and the unordered list is not. This is an issue with the markdown parser itself, you can see maybe if the blog content links can be identified separately and then you can underline them by using simple CSS, but if you can't, we will look for some other way to do this.

No... No.... It does not like this. Actually I did what you suggested that to show links in lists. That bug, which @pranshukharkwal  told solved, but then this issue appears. That all the links in the main content becomes like this.
*It is actually a CSS Bug* 
So what can I do.... 
the <a> tags inside main content have the underline, this will make all the ordered and unordered lists links underlined.
So what I am saying is... it does not look good as I shown in the above picture.
